### DISCUSSION TITLE
Some features I hope we have in ToonSquid 2.0 (Or earlier) :D

### DISCUSSION THREAD

**Discussion by DinoAwakening** [keiwando/toonsquid#95] [Ideas]
While using this app, I thought about some features that would make this app unstoppable, and superior to any other app on mobile (**Perhaps, even being compared to PC software**) :D  

-Customizable HUD (Change location of tools, layers and timeline).
-Magic selection by color (Something similar to MediBang Paint).
-Easily set the duration of the drawings (Change the "_Expand_" button to something like "**- (Modifiable number) +**").
-Have an opacity bar (similar to the previous feature) in drawing and animation capability.
-When you Warp with Pivot, the axis is not seen in some scenarios. The pivot could be white with black borders, and vice versa (Or even directly, negative color from the one shown at the moment).
-The smooth of bitmap brushes is strange. Making them like the vector brush smooth would be more satisfactory.
-Clipboard of copied items by double-clicking on the lasso tool. That way, you can copy individual selections to other layers.

That's quite a few things to ask for, but I feel it would make ToonSquid the best experience compared to the rest of the animation apps. I hope Keiwan sees this! :D

**Comment by keiwando** [comment]
Thank you for all of your suggestions!

> Have an opacity bar (similar to the previous feature) in drawing and animation capability.

Would you mind elaborating on this a bit? I understood the previous feature but I'm not quite sure on this one. Are you asking for an opacity slider for an entire drawing, in addition to the opacity sliders for the drawing layer and animation layers that currently exist?

> the axis is not seen in some scenarios

Which axis / pivot are you referring to here? The layer pivot should not be visible when you are warping layers. Can you maybe post a screenshot?

**Reply by DinoAwakening** [reply]
Hold on, I’m too excited cause you did actually read my ideas! I’ll make them real concepts entirely just using ToonSquid! :D

**Reply by DinoAwakening** [reply]
There you go! :D  

https://youtu.be/i2ZlolZOH98

Pstd: Forget about opacity bar, these are the functions we actually need :)

**Reply by keiwando** [reply]
Thanks for taking the time to make a video for those suggestions, I appreciate it!

**Comment by DinoAwakening** [comment]
2/7 Features were achieved on 1.9. That’s fine though! Nice update you’ve made Keiwando :D
